src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static Random random = new Random();
+
+    public static Random getRandom() {
```

Comment:
This class is a color supplier - it shouldn't provide `Random` instance as it violates Single Responsibility Principle.

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/68182494/5675dbb6-2d4e-4ec5-9088-bcecfca3306b)
created new class called Random (may be it could be better to use Java API class , but i think that this cleaner for people to understand code)